README.md

```diff
@@ -109,3 +109,14 @@ Example of running Cloudy independently:
 
 ## Output params:
 * Lines in ```data/LineLabels_filtered.txt```.
+
+
+## Testing
+Tests can be found in ```tests/```, to run the tests with pytest:
+1. Install pytest ```pip install pytest```
+2. Navigate to tests ```cd tests```
+3. Run tests ```pytest -v test.py```, the -v flag is optional for verbosity
+
+The tests check if cloudy path in ```src/common/settings.py``` is the correct path of the cloudy installation in our system by runnning a test model.in, then checks if the output of CLOUDY is ok ("Cloudy exited OK"), then creates the file structure src.manager.QueueManager() expects and runs 4 models on multiple cpus to check the pipeline.
```

Comment:
Just a thought: If the settings.py file is supposed to be edited by the user, maybe we should not bury it so deep in the file tree.